independent-publisher-connectors/Proposal - jenga (Independent Publisher)/README.md

```diff
@@ -0,0 +1,3 @@
+# JENGA
```

Comment:
Hi @Bkaraba,
Thank you so much for your hard work with us.
If you would like us to move forward with the "24 pull requests" connector, please remove the other Proposal "Jenga" file.
Please let us know if you have any questions.

Reply:
fixed